contracts.md

```diff
@@ -64,3 +64,8 @@ due to historical reasons.
 ## Treasury
 
 https://etherscan.io/address/0x886CE997aa9ee4F8c2282E182aB72A705762399D
+
+The treasury contract is a simple vault implementation holding all RFV funds of
```

Comment:
```suggestion
The treasury contract is a simple vault implementation holding all the funds collected by
```

Reply:
The reason behind this change is that in addition to DAI, the treasury also holds the full amount of SLP token, not just its RFV.